devices/tuya.js

```diff
@@ -1782,4 +1782,14 @@ module.exports = [
             exposes.enum('clear_device_data', ea.SET, ['clear']),
         ],
     },
+    {
+        fingerprint: [{modelID: 'TS004F', manufacturerName: '_TZ3000_4fjiwweb'}],
+        model: 'ERS-10TZBVK-AA',
+        vendor: 'TuYa',
+        description: 'Smart knob',
+        fromZigbee: [fz.command_step, fz.command_toggle, fz.command_move_hue],
+        toZigbee: [],
+        exposes: [e.action(['button_*_hold', 'button_*_single', 'button_*_double', 'button_*_triple', 'button_*_quadruple',
+            'button_*_release'])],
```

Comment:
Please replace the `*` with the actual values the are being published as `action`